client/src/app/site/pages/meetings/pages/agenda/modules/topics/pages/topic-detail/components/topic-detail/topic-detail.component.scss

```
@@ -27,3 +27,14 @@
     color: green;
     margin-top: 8px;
 }
+
+.attachment-entry {
+    height: auto;
+    height: max-content !important;
```

Is this really needed? 

```suggestion
    height: max-content !important;
```